backend/migrations/20250503021135-create-student_events-table.js

```diff
@@ -0,0 +1,35 @@
+export async function up(queryInterface, Sequelize) {
+  await queryInterface.createTable('StudentEvents', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER,
+    },
+    student_id: {
+      type: Sequelize.STRING,
+      alllowNull: false,
+    },
+    meeting_id: {
+      type: Sequelize.STRING,
+      allowNull: false,
+    },
```

Comment:
Since these two fields should be foreign keys, you'll need to reference the tables here as well

See the example near the end of this page: https://sequelize.org/docs/v6/other-topics/migrations/